.github/workflows/tests.yaml

```diff
@@ -513,6 +514,7 @@ jobs:
                   echo "NETWORK_MANAGEMENT_APP: out/linux-x64-network-manager-ipv6only-no-ble-no-wifi-tsan-clang-test/matter-network-manager-app" >> /tmp/test_env.yaml
                   echo "FABRIC_ADMIN_APP: out/linux-x64-fabric-admin-rpc-ipv6only-no-ble-no-wifi-clang/fabric-admin" >> /tmp/test_env.yaml
                   echo "FABRIC_BRIDGE_APP: out/linux-x64-fabric-bridge-rpc-ipv6only-no-ble-no-wifi-clang/fabric-bridge-app" >> /tmp/test_env.yaml
+                  echo "FABRIC_SYNC_APP: out/linux-x64-fabric-sync-ipv6only-no-ble-no-wifi-clang/fabric-sync" >> /tmp/test_env.yaml
```

Comment:
any chance we can also add this to https://github.com/project-chip/connectedhomeip/blob/master/scripts/tests/local.py#L166 and https://github.com/project-chip/connectedhomeip/blob/master/scripts/tests/local.py#L384 ?

at some point we may want to consolidate these to be one script as local is intended to "build what is needed" and "able to run all tests possible"

Reply:
Done